chain/params.go

```diff
@@ -9,6 +9,7 @@ type Params struct {
 	Forks          *Forks                 `json:"forks"`
 	ChainID        int                    `json:"chainID"`
 	Engine         map[string]interface{} `json:"engine"`
+	Whitelists     map[string]interface{} `json:"whitelists,omitempty"`
```

Comment:
(Question:) What's wrong with having this field not being `ompitempty` ?

Reply:
If you create genesis file without any whitelist, value is going to be nil.